src/modules/networking/src/lib/Networking.cpp

```diff
@@ -31,6 +31,9 @@ const char* g_getIpAddressDetails = "ip addr";
 const char* g_getDefaultGateways = "ip route";
 const char* g_getDnsServers = "systemd-resolve --status";
 
+const char* g_checkSystemdResolvedIsRunning = "systemctl list-units --type=service --state=running | grep systemd-resolved.service";
+const char* g_startSystemdResolved = "systemctl start systemd-resolved.service";

```

Comment:
Don't do this. Instead, use the following commonutils library functions:

bool IsDaemonActive(const char* name, void* log)

And:

bool EnableAndStartDaemon(const char* name, void* log)

Reply:
You can directly call just the later one, EnableAndStartDaemon.

Reply:
Thanks! This is a great improvement. Addressed.